### DISCUSSION TITLE
【问题反馈】关于下载功能的回显问题

### DISCUSSION THREAD

**Discussion by xialibaixue** [xiaoyaocz/flutter_dmzj#88] [General]
![Screenshot_20230314_222035](https://user-images.githubusercontent.com/77116861/225031622-74564a9c-fd31-435b-a12e-304677a124a8.jpg)
如图中所示，我使用下载功能下载了几章漫画，但是下载的显示同时出现在了漫画和小说的位置上。

**Comment by xiaoyaocz** [comment]
https://github.com/xiaoyaocz/flutter_dmzj/commit/51cc61d284b8da43a9fda34882ed7949ed49b46f
